src/utilities/include/par.h

```diff
@@ -598,13 +598,13 @@ void transform(ExecutionPolicy policy, InputIter first, InputIter last,
       "You can only parallelize RandomAccessIterator.");
 #if MANIFOLD_PAR == 'T'
   if (policy == ExecutionPolicy::Par) {
-    tbb::parallel_for(tbb::blocked_range<size_t>(
-                          0_z, static_cast<size_t>(std::distance(first, last))),
-                      [&](const tbb::blocked_range<size_t> &range) {
-                        std::transform(first + range.begin(),
-                                       first + range.end(),
-                                       d_first + range.begin(), f);
-                      });
+    tbb::parallel_for(
+        tbb::blocked_range<size_t>(
+            0_uz, static_cast<size_t>(std::distance(first, last))),
+        [&](const tbb::blocked_range<size_t> &range) {
+          std::transform(first + range.begin(), first + range.end(),
+                         d_first + range.begin(), f);
+        });
```

Comment:
I believe this can be
```cpp
    tbb::parallel_for(
        tbb::blocked_range(first, last),
        [&](const auto& range) {
          std::transform(range.begin(), range.end(),
                         d_first + std::distance(first, range.begin()), f);
        });
```